samples/text_panes/components/text_pane.js

```diff
@@ -0,0 +1,97 @@
+AFRAME.registerComponent("text_pane", {
+  schema: {
+    jsonData: {
+      parse: JSON.parse,
+      stringify: JSON.stringify,
+    },
+  },
+
+  multiple: true,
+
+  init: function () {
+    const data = this.data.jsonData;
+    const el = this.el;
+
+    // Create textLabel
+    this.textLabel = document.createElement("a-text");
+    this.textLabel.setAttribute("id", "text");
+    this.textLabel.setAttribute("value", data.text[data.currPosition]);
+    this.textLabel.setAttribute("negate", "true");
+    this.textLabel.setAttribute("scale", "2 2 1");
+    this.textLabel.setAttribute("position", "-4.9 0 0.25");
+    this.el.appendChild(this.textLabel);
+
+    if (data.text.length > 1) {
+      const leftNavProp =
+        '{"width": "1.5", "height": "1.5", "depth": "0.01", "color": "white", "x": "-8", "y": "0", "z": "0.005"}';
+      const rightNavProp =
+        '{"width": "1.5", "height": "1.5", "depth": "0.01", "color": "white", "x": "8", "y": "0", "z": "0.005"}';
+
+      // Create left nav textLabel
+      this.buttonLabelLeft = document.createElement("a-text");
+      this.buttonLabelLeft.setAttribute("id", "text-nav-left");
+      this.buttonLabelLeft.setAttribute("value", "Prev");
+      this.buttonLabelLeft.setAttribute("negate", "true");
+      this.buttonLabelLeft.setAttribute("scale", "2 2 1");
+      this.buttonLabelLeft.setAttribute("color", "black");
+      this.buttonLabelLeft.setAttribute("position", "-8.5 0 0.01");
+      this.el.appendChild(this.buttonLabelLeft);
+
+      // Create left nav button
+      this.leftNav = document.createElement("a-entity");
+      this.leftNav.setAttribute("id", "nav-button-left");
+      this.leftNav.setAttribute("prism", "jsonData", leftNavProp);
+      this.leftNav.addEventListener("click", function () {
+        if (data.currPosition != 0) {
+          --data.currPosition;
+          document
+            .querySelector("#text")
+            .setAttribute("value", data.text[data.currPosition]);
+          document
```

Comment:
Is there a reason for this part here? The part about `.setAttribute("jsonData", rightNavProp);` for the `#text-nav-right`?

Reply:
I think we want to reset the text to be `Next` here right? Right now going all the way to the end, it is `Done`, but doesn't reset to `Next` on going left

Reply:
Oops yea should be reset to next when the left nav button is clicked, should be fixed now